aiida/storage/psql_dos/migrations/utils/legacy_workflows.py

```diff
@@ -9,13 +9,13 @@
 ###########################################################################
 # pylint: disable=invalid-name
 """Utilities for removing legacy workflows."""
+import json
```

Comment:
Have you checked the migration tests still work? (given these tests do not run in PRs any more)

Lower in this file, it passes a 'wb' handle to `json.dump`, which I assume will fail?

Reply:
Good point, but I think even the tests wouldn't catch this because I think that path of the code is uncovered. Will have a look later.

Reply:
should catch it, I added a test specifically for it in #5330

Reply:
> I added a test specifically for it in

Fantastic, can confirm that it failed, well spotted! I have fixed it and migration tests now run fine. Also removed some remaining uses of `aiida.common.json` in the tests. Should be good to go now

Reply:
cheers 👍 will have a look in a bit